phac-nml/irida-next
# Workflow Submission: Add namespace_id to workflow model

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Adds a parameter `namespace_id` to the workflow execution params.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/11295750/1a3201c6-2269-494e-8ceb-54dadb35e0bb)

![image](https://github.com/phac-nml/irida-next/assets/11295750/887d185d-7076-47b5-a73e-74be674138ec)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Login and go to a project that you can execute a workflow from
2. Open developer tools, go to the network tab and select persist logs
3. Select a pipeline and submit
4. In network tab, look at the logs and you should see something similar to: 
![image](https://github.com/phac-nml/irida-next/assets/11295750/794ac38b-86cd-4d78-afb3-45c11370a7e6)

Check the rails console after executing a pipeline:
1. Inspect the workflow you just ran: `WorkflowExecution.last`
2. The `namespace_id` should be added to the workflow_execution 

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
